### DISCUSSION TITLE
Is it possible to redirect to the requested page after login?

### DISCUSSION THREAD

**Discussion by VenomXL** [spatie/laravel-login-link#10] [Q&A]
The package works great but is it possible to redirect to the requested page after user is authenticated and logged in successfully?
In function `getRedirectUrl()` if we could just implement `return redirect()->intended()->getTargetUrl();` instead of `return "/";` would actually solve this issue.

**Comment by freekmurze** [comment]
That is currently not supported, but feel free to create a PR that adds this functionality. Make sure to update the readme and tests.
